src/ui/color/ColorHelper.js

```diff
@@ -21,7 +21,7 @@
  *****************************************************************************/
 
 export const COLOR_PALETTE = [
-    [0x00, 0x37, 0xFF],
+    [0x43, 0xB0, 0xFF],
```

Comment:
I'm not an official reviewer, but this just looks weird. Is there any reason we are storing the color palette as an array of hex values opposed to anything else (Color objects, int array, hexstrings, rgb string)? I feel like this is unintuitive for most developers

Reply:
Agree, I have no idea why we did it this way. Seems like a simpler array of hex strings would be more dlrect.